src/data/dynamic/form/Interest.js

```diff
@@ -29,9 +29,26 @@ export const FIELDS = {
     editable: false,
     required: true,
   },
+  requirements: {
+    text: "Terms and Conditions",
+    input: "terms",
+    width: 12,
+    field: "requirements",
+    options: [
+      "I have read the MLH code of conduct and agree to the terms and conditions listed",
+      "I consent to photographs being taken and being used for future marketing purposes",
+      "I consent to providing a safe space for hackers to learn and grow their interests in computing",
+      "I consent to following the provided guidelines and rules instructed by the organizing team",
+      "I understand that failure to comply with guidelines or creating an unsafe space will result in my removal from the event",
+      "I understand this is an in person event taking place in UCR and I must attend in person in order to participate",
+      "I understand that I will be given access to private data and malicious intents and actions will be reported immediately.",
```

Comment:
this is only for admin, make sure to refer to the participant terms and conditions